### PR TITLE
Feature lwdev 5888 settings api integration

### PR DESCRIPTION
**Pull request by mihail-yartsev**

### CODE REVIEW

File: src/MarginTrading.OrderbookAggregator/Infrastructure/Implementation/SystemService.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace MarginTrading.OrderbookAggregator.Infrastructure.Implementation
+{
+    internal class SystemService : ISystem
```
Comment (GSailen) [inline src/MarginTrading.OrderbookAggregator/Infrastructure/Implementation/SystemService.cs]: Quite strange.. what is it for ?

File: src/MarginTrading.OrderbookAggregator/Infrastructure/Implementation/TelemetryService.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections.Generic;
+using Microsoft.ApplicationInsights;
+
+namespace MarginTrading.OrderbookAggregator.Infrastructure.Implementation
+{
+    public class TelemetryService : ITelemetryService
```
Comment (GSailen) [inline src/MarginTrading.OrderbookAggregator/Infrastructure/Implementation/TelemetryService.cs]: Not used

File: src/MarginTrading.OrderbookAggregator/Modules/OrderbookAggregatorModule.cs [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        /// <summary>
+        /// Scans for types in the current assembly and registers types which: <br/>
+        /// - are named like 'SmthService' <br/>
+        /// - implement an non-generic interface named like 'ISmthService' in the same assembly <br/>
+        /// - are the only implementations of the 'ISmthService' interface <br/>
+        /// - are not generic <br/><br/>
+        /// Types like SmthRepository are also supported.
+        /// Also registers startup for implementations of <see cref="ICustomStartup"/>.
+        /// </summary>
+        private void RegisterDefaultImplementations(ContainerBuilder builder)
```
[the remark this answers is not in the document]
Comment (mihail-yartsev) [reply src/MarginTrading.OrderbookAggregator/Modules/OrderbookAggregatorModule.cs]: Is's a rather common approach to register via reflection.

File: src/MarginTrading.OrderbookAggregator/Infrastructure/Implementation/SystemService.cs (hunk 1 above)
Comment (mihail-yartsev) [reply src/MarginTrading.OrderbookAggregator/Infrastructure/Implementation/SystemService.cs]: This was meant to isolate dependency on the DateTime.UtcNow - for tests.

**Pull request merged**